### PR TITLE
Migrate README file from the ruleset repository to the ruleset folder.

### PR DESCRIPTION
**Pull request by palaciosjeremias**
|Related issue|
|---|
|6939|

This PR migrates the original README file from the **Ruleset** repository to the new ruleset folder and adapts its content with new descriptions.

### CODE REVIEW

File: ruleset/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+[![Slack](https://img.shields.io/badge/slack-join-blue.svg)](https://goo.gl/forms/M2AoZC4b2R9A9Zy12)
+[![Email](https://img.shields.io/badge/email-join-blue.svg)](https://groups.google.com/forum/#!forum/wazuh)
+[![Documentation](https://img.shields.io/badge/docs-view-green.svg)](https://documentation.wazuh.com)
+[![Documentation](https://img.shields.io/badge/web-view-green.svg)](https://wazuh.com)
+
+Wazuh ruleset is used to detect attacks, intrusions, software misuse, configuration problems, application errors, malware, rootkits, system anomalies or security policy violations.
+
+The ruleset includes compliance mapping with PCI DSS v3.1 and CIS.
+
+## Installation
```
Comment (pereyra-m) [inline ruleset/README.md:12]: I would remove the "Installation" section, there is no installation nor update process related to the ruleset now.

File: ruleset/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Full documentation at [documentation.wazuh.com](https://documentation.wazuh.com/current/user-manual/ruleset/index.html)
+
+## Contribute
+
+If you have created new rules, decoders or rootchecks and you would like to contribute to our repository, please fork our Github repository and submit a pull request. To make a pull request for new rules and decoders, follow these instructions:
+
+1. If your rules and decoders are related to existent ones in the ruleset, you should add them at the end of the corresponding file. If they are made for a new application or device that Wazuh does not currently support, you should create a new `XML` following the title format. For example, if the last `XML` file is `0620-last-xml_rules.xml`, the next one should be named `0625-new_integration.xml`. Please, make sure your rules do not use an existent `rule id`.
+
+2. Make sure to create your `test.ini` file. You may find examples under the `wazuh/wazuh-ruleset/tools/rules-testing/tests` folder. Then add it to the repository along with the rest of the tests.
```
Comment (pereyra-m) [inline ruleset/README.md:39]: This reference to `wazuh-ruleset` should be modified

**Review by pereyra-m — APPROVED** [review]
LGTM!

File: ruleset/README.md (hunk 1 above)
Comment (DProvinciani) [reply ruleset/README.md:12]: +1

File: ruleset/README.md (hunk 2 above)
Comment (DProvinciani) [reply ruleset/README.md:39]: +1

**Review by DProvinciani — CHANGES_REQUESTED** [review]
Please, apply changes according to the @pereyra-m comments.

File: ruleset/README.md (hunk 1 above)
Comment (palaciosjeremias) [reply ruleset/README.md]: You are right! Fixed.

File: ruleset/README.md (hunk 2 above)
Comment (palaciosjeremias) [reply ruleset/README.md]: You are right! Fixed.

**Review by DProvinciani — APPROVED** [review]
Looks good!

**Pull request merged**